src/ralph/data_center/tests/test_api.py

```diff
@@ -53,7 +55,7 @@ def setUp(self):
 
     def test_get_data_center_assets_list(self):
         url = reverse('datacenterasset-list')
-        with self.assertNumQueries(15):
+        with self.assertNumQueries(21):
```

Comment:
why number of queries has grown? Are you sure that it wouldn't grow linearly with more `DataCenterAsset`s on the list?

Reply:
That was because of using filter in serialiser, where it fetched data from database for each asset in the list.